### PR TITLE
Cleanup 6

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

Cleanup + Ports:

- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/29.
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/33
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/32
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/35
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/38
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/40
- [x] - https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/42

non-null chance of missing stuff.

### CODE REVIEW

**Comment by Kevak** [comment]
Do not remove beams. We use those for various spells. Nix the junglemobs from the same ported pr tho. We don't use those. I intend to do a refactor for beams sometime in the next month.

**Comment by Kevak** [comment]
Similarly you should give that beam PR a go-over. It looks like it's nixing a lotta status effects from projectilecode that it really shouldn't be nixing.

**Comment by Kevak** [comment]
Yeah further looking through it. Good few things we do use that we should not be nixing and will break shit if we remove. I really must voice the question. _Why_ exactly, have we been blind porting all these removals without thorough review and testing? Just the other day a zlevel completely broke due to it. Methinks either the time to stop has come or the time to take proper care with what we remove from our codebase has come, rather than blindporting. One of the two.

**Comment by Gyrandola** [comment]
> Yeah further looking through it. Good few things we do use that we should not be nixing and will break shit if we remove. I really must voice the question. _Why_ exactly, have we been blind porting all these removals without thorough review and testing? Just the other day a zlevel completely broke due to it. Methinks either the time to stop has come or the time to take proper care with what we remove from our codebase has come, rather than blindporting. One of the two.

The missing zlevel is unrelated.
All this "blindporting" has only caused less than a handful of minor issues that got fixed a day later, in exchange for a much cleaner codebase.

**Comment by Kevak** [comment]
That is fair I suppose

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -11,11 +11,6 @@
 	flags_1 = null
 	slot_flags = ITEM_SLOT_BELT
 	possible_item_intents = list(INTENT_GENERIC)
-	custom_materials = list(/datum/material/iron=50, /datum/material/glass=20)
-	light_system = MOVABLE_LIGHT
```
Comment (TheFenix323) [inline code/game/objects/items/devices/flashlight.dm:15]: Removing this is what causes #1821 to happen

File: code/game/objects/items/soap.dm
Comment (TheFenix323) [inline code/game/objects/items/soap.dm:1]: The new soap is missing the cleaning code, causing #1824 to happen

**Review by Cyprex — APPROVED** [review]
sure

**Comment by Githubaccount222** [comment]
Wish miner's salve wasn't removed. It's one of the few things we can actually make.

**Pull request merged**